File: help/pt_pt/exercicios_da_ficha_4/Ficha04_002/Ficha04_002.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Primeiro valor \t: " );// print text
    n1 = parseInt(prompt("Primeiro valor \t: " ,"0") );
    
    document.write(n1 + "\n");// print input 
    document.write("Segundo valor \t\t: " );// print text
    n2 = parseInt(prompt("Segundo valor \t\t: " ,"0") );
    
    document.write(n2 + "\n");// print input 
    var igual = n1 == n2 ;
    document.write("\n" + n1 + "\t==\t" + n2 + "\t=\t" + igual );
    var diferente = n1 != n2 ;
    document.write("\n" + n1 + "\t!=\t" + n2 + "\t=\t" + diferente );
    var compara = n1 > n2 ;
    document.write("\n" + n1 + "\t>\t" + n2 + "\t=\t" + compara );
    compara = n1 >= n2 ;
    document.write("\n" + n1 + "\t>=\t" + n2 + "\t=\t" + compara );
    document.write("\n" + n1 + "\t<\t" + n2 + "\t=\t" + ( n1 < n2 ) );
    document.write("\n" + n1 + "\t<=\t" + n2 + "\t=\t" + ( n1 <= n2 ) );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:30

File: help/pt_pt/exercicios_da_ficha_4/Ficha04_002/Ficha04_002.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha04_002 {

    //Main Function
    static void Main() {
        Console.Write("Primeiro valor \t: " );
        int n1 = int.Parse(Console.ReadLine());
        Console.Write("Segundo valor \t\t: " );
        int n2 = int.Parse(Console.ReadLine());
        bool igual = n1 == n2 ;
        Console.Write("\n" + n1 + "\t==\t" + n2 + "\t=\t" + igual );
        bool diferente = n1 != n2 ;
        Console.Write("\n" + n1 + "\t!=\t" + n2 + "\t=\t" + diferente );
        bool compara = n1 > n2 ;
        Console.Write("\n" + n1 + "\t>\t" + n2 + "\t=\t" + compara );
        compara = n1 >= n2 ;
        Console.Write("\n" + n1 + "\t>=\t" + n2 + "\t=\t" + compara );
        Console.Write("\n" + n1 + "\t<\t" + n2 + "\t=\t" + ( n1 < n2 ) );
        Console.Write("\n" + n1 + "\t<=\t" + n2 + "\t=\t" + ( n1 <= n2 ) );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Mon, 2022 Sep 26 15:30
